### PR TITLE
CMS-8522 - Sync wrapper with site links

### PR DESCRIPTION
**Pull request by ashbru**
**PURPOSE**
There is a need to update the wrapper to match the header, mast, and footer on the site.

**TESTING**
1.  Clone this repo if it's the first time.

2.  Checkout the change branch

3.  Install all packages - `npm i`

4.  Build Code - `npm run build`

5.  Run Server - `npm run start`

6.  Open a browser tab and navigate to http://localhost:8080/

7.  Verify that your changes match the site (https://www.dallasnews.com/)

image-20240112-201529
image-20240112-201609

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,3 +59,19 @@ I used John Hancock's Yeoman generated starter project to implement an example f
...
+
+3.  Install all packages - `npm i`
+
+4.  Build Code - `npm run build`
+
+5.  Run Server - `npm run start`
+
+6.  Open a browser tab and navigate to http://localhost:8080/
+
+7.  Verify that your changes match the site
```
Comment (nigelgilbert) [inline README.md:77]: Thank you for the docs TLC, sir 🙏

**Review by nigelgilbert — APPROVED** [review]
LGTM!

Diff is pretty minimal.  The markup looks correct.  I ran it locally and copied the pages' HTML markup into an HTML validator.

Looks good!

File: src/index.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
```
Comment (nigelgilbert) [inline src/index.html:4]: Hey, the HTML validator was yelling about these 2 small things!
 
 Super minimal and totally unrelated to your changes, so I just fixed 'em real quick.
 
 Hopefully that's cool.  Just wanted to make things easy.
 
These bugs were there from the begining. Good work w/ the PR.

**Review by sihameelati — COMMENTED** [review]
Thank you for adding the testing steps 👍  Could you please update the copyright to 2024 on the footer

File: README.md [hunk 3]
```diff
@@ -59,3 +59,19 @@ I used John Hancock's Yeoman generated starter project to implement an example f
 ```
 
 - change that link to the new promo offer, e.g. `dn-header-08042021`
+
+## Testing
+
+1.  Clone this repo if it's the first time.
+
+2.  Checkout the change branch
```
Comment (xtinarees) [inline README.md:67]: this is a nitpick but I don't think these 2 steps are necessary to include -- they're a given. Love the overall addition of docs!

**Pull request merged**